### DISCUSSION TITLE
Do I need rust dependencies if I'm not going to use rust?

### DISCUSSION THREAD

**Discussion by lianhuilui** [LunarVim/LunarVim#4241] [Q&A] [answered]
I've just run the installer script and the script asked to install npm, pip, and then rust dependencies.

The script did not explained if they are needed, (the default answer was "no") So my main question is what are they for and if I need them if I'm not going to write rust projects.

The dependencies from the script:
```sh
declare -a __rust_deps=(
  "fd::fd-find"
  "rg::ripgrep"
)
```

**Comment by LostInTheLogs** [comment, marked as answer]
fd and ripgrep are suggested dependencies for https://github.com/nvim-telescope/telescope.nvim/, if you don't install them you'll need to configure telescope differently
